### PR TITLE
Switch the model to evaluation mode before generation in the language model example

### PR DESCRIPTION
**Pull request by Henderake**
The model in the word language model example uses dropout, so I add a line to call model.eval() before generation.

### CODE REVIEW

**Comment by soumith** [comment]
thanks!
